app/src/main/res/layout/fragment_forgot_password.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This could be simplified to a `LinearLayout`

Reply:
I assumed you change because of the background images. Let's discuss how that can be solved